advanced_functionality/autogluon-tabular/AutoGluon_Tabular_SageMaker.ipynb

```diff
@@ -0,0 +1,627 @@
+{
```

Comment:
<p>Is this example expected to be run in CN regions? If so, please update the s3 endpoint domain to change based on regions.</p><p> </p><p>"amazonaws.com" won't work if the bucket is in "cn-north-1" or "cn-northwest-1", instead it should be "amazonaws.com.cn" in such cases.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1129/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Changed the method to handle the uri